test/nbrowser/AccessRules1.ts

```diff
@@ -202,6 +202,10 @@ describe('AccessRules1', function() {
     await driver.find('.test-tools-access-rules').click();
     await driver.findWait('.test-rule-set', 2000);
     const ruleSet = findDefaultRuleSet(/FinancialsTable/);
+    await driver.executeScript(
```

Comment:
There is `gu.scrollIntoView(ruleSet)` (but it's not identical, so if you need a different version intentionally, that's fine).